packages/web/src/pages/sign-up-page/pages/SelectArtistsPage.tsx

```diff
@@ -99,6 +102,17 @@ export const SelectArtistsPage = () => {
 
   const ArtistsList = isMobile ? Flex : Paper
 
+  const styles = useSpring({
```

Comment:
rather than integrate this page with Page, simply duplicating the transition styles for now.

Reply:
Is the reasoning just for scope purposes? Or is there a specific reason this shouldn't be using `Page`

Reply:
i have avoided adding page to this page cause it's pretty weird/hard and i'd rather focus on the other bugs basically. could be something to circle back on though!